<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="FilterLabel">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QGroupBox" name="fareGroupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Price</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="fareFromLabel">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="fareFromMinLabel_2">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="fareFromSlider">
            <property name="maximum">
             <number>27000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="fareFromMaxLabel">
            <property name="text">
             <string>28000</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fareToLabel">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="fareToMinLabel">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="fareToSlider">
            <property name="maximum">
             <number>28000</number>
            </property>
            <property name="value">
             <number>28000</number>
            </property>
            <property name="sliderPosition">
             <number>28000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="fareToMaxLabel">
            <property name="text">
             <string>28000</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="brandFilter">
      <item>
       <property name="text">
        <string>Brand</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QComboBox" name="colorFilter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Color</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="sexFilter">
      <item>
       <property name="text">
        <string>Sex</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2" rowspan="8" colspan="3">
     <widget class="QTableView" name="GarmentTable"/>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="sizeFilter">
      <item>
       <property name="text">
        <string>Size</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="seasonFilter">
      <item>
       <property name="text">
        <string>Season</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="nameFilterlineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>                      search by SKU</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="CompanyLabel">
      <property name="font">
       <font>
        <pointsize>27</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>The Garment</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnAccount">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Account</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="categoryFilter">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Category</string>
      </property>
      <item>
       <property name="text">
        <string>Category</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="loadButton">
      <property name="text">
       <string>Load File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
